File: ProfilesApi/Contracts/Responses/ReceptionistProfiles/GetDetailedReceptionistProfilesResponse.cs
namespace ProfilesApi.Contracts.ReceptionistProfiles;

public class GetDetailedReceptionistProfilesResponse
{
    public Guid Id { get; set; }
    public string FirstName { get; set; }
    public string LastName { get; set; }
    public string? MiddleName { get; set; }
    public Guid AccountId { get; set; }
    public Guid OfficeId { get; set; }
    public string Address { get; set; }
}

File: serverApp/src/app/_models/responses/GetDetailedReceptionistProfilesResponse.ts
import { Guid } from "guid-typescript";

export class GetDetailedReceptionistProfilesResponse {
    constructor (
      public id: Guid,
      public email: string,
      public firstName: string,
      public lastName: string,
      public middleName: string | null,
      public address: string | null,
      public officeId: Guid,
      public accountId: Guid,
      public Role: string,
    ) {}
  }